### PR TITLE
docs: ✏️ introducing PR template

### PR DESCRIPTION
**Pull request by annsch**
- introducing ne PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- [ ] Lint and unit tests pass locally with my changes
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added necessary documentation (if appropriate)
+
+## Further comments
+
+If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
+
+💔Thank you!
```
Comment (mfranzke) [inline .github/PULL_REQUEST_TEMPLATE.md:27]:
```suggestion
❤️ Thank you!
```

**Review by mfranzke — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
```
Comment (mfranzke) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Types of changes
+
+What types of changes does your code introduce?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
```
Comment (nmerget) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: Maybe we can add `refactor` here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Checklist
```
Comment (nmerget) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: Do we need the checklist?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (annsch) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (annsch) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: I remove the linter item. I'd suggest to leave the others. It's a reminder not to forget documentation and tests.

Comment (nmerget) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: But if you change for example the documentation, you would not test it, right?

Comment (annsch) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: ok, convinced 👯

Comment (annsch) [reply .github/PULL_REQUEST_TEMPLATE.md]: @nmerget fixed this

**Review by nmerget — APPROVED** [review]

**Pull request merged**